src/gui/modals/ExportProjectDialog.cpp

```diff
@@ -102,6 +99,32 @@ ExportProjectDialog::ExportProjectDialog( const QString & _file_name,
 			this, SLOT(startBtnClicked()));
 }
 
+void ExportProjectDialog::updateWindowTitle()
+{
+	setWindowTitle(tr("Export project to %1").arg(m_targetFileInfo.fileName()));
+}
+
+QString ExportProjectDialog::removeKnownExtension(QString filePath)
+{
+	QFileInfo info(filePath);
+
+	QString fileExt = info.suffix();
+	
+	if (fileExt.isEmpty()) { return filePath; }
+	
+	fileExt = "." + fileExt;
+
+	for (int i = 0; i < ProjectRenderer::NumFileFormats; i++)
```

Comment:
``ProjectRenderer::fileEncodeDevices`` is of type ``std::array``, so we should be able to iterate it using a ranged-based for loop. I would prefer that as we can use a variable that's simply named  ``device`` instead of the verbose ``ProjectRenderer::fileEncodeDevices[i]``.